dev/release/utils-prepare.sh

```diff
@@ -121,18 +121,15 @@ update_versions() {
     DESCRIPTION
   rm -f DESCRIPTION.bak
   git add DESCRIPTION
+  # Replace dev version with release version
```

Comment:
I think there's another gap here with regards to patch releases since that version bump goes on top of the previous major release branch. It's fine as long as there is a news entry corresponding to the patch release before the script is run, but if not, this will overwrite the current release version entry. I'm not sure there's a great solution to this.

Reply:
I didn't confirm carefully but this will work well by running the following command on `maint-X.Y.x` branch after we release X.Y.0:

```bash
BUMP_DEFAULT=0 BUMP_VERSION_POST_TAG=1 dev/release/post-12-bump-versions.sh X.Y.0 X.Y.1
```